compiler/src/iree/compiler/Codegen/Common/DecomposePackUnPackOps.cpp

```diff
@@ -56,7 +56,8 @@ struct LowerPackPattern : public OpRewritePattern<tensor::PackOp> {
     if (controlFn && failed(controlFn.value()(op))) {
       return failure();
     }
-    FailureOr<linalg::LowerPackResult> res = linalg::lowerPack(rewriter, op);
+    FailureOr<linalg::LowerPackResult> res =
+        linalg::lowerPack(rewriter, op, false);
```

Comment:
could add something like `/*lowerPadLikeWithInsertSlice=*/false` so that the reader knows what this flag is.

Reply:
Nice catch, all review comments are addressed in next commit. Thanks!